Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM python:3
+FROM python:3-stretch
 
 LABEL maintainer="contact@zooniverse.org"
 
 RUN apt-get update \
   && apt-get install -y --no-install-recommends \
     postgresql-client \
     gdal-bin \
-    libtiff5-dev \
+    libtiff-dev \
```

Comment:
Make sure you're looking at the correct debian version when looking up packages, this base images is debian stretch, not jessie. https://www.debian.org/releases/

that said you are correct, `libtiff-dev` is a virtual package which installed the current library for the OS version, in this case `libtiff5-dev` https://packages.debian.org/stretch/libtiff-dev

It might be best to lock it to the more specific version but doing so means we don't get the OS lib updates which can be good. The pylibtiff lib seems to be slow moving, in theory the newer versions of libtiff-dev are supported at least on master branch, https://github.com/pearu/pylibtiff/issues/107#issuecomment-583852384

Though i failed to get it working with them even with master checked out for pylibtiff

Reply:
Those OS lib updates might also break this incredibly fragile ecosystem of kernels and libs and modules and images that you've gotten to work, so it seems like more specific would be better. If this updates to `libtiff6-dev` I can't imagine that it'll keep working. I don't see a benefit to ambiguity here or with the base image version.

Reply:
```suggestion
    libtiff5-dev \
```